flask_saml2/sp/sp.py

```diff
@@ -49,7 +49,9 @@ def login_successful(
         but they *must* call ``super()``.
         """
         self.set_auth_data_in_session(auth_data)
-        return redirect(relay_state)
+        if not redirect_to:
+            redirect_to = self.get_login_return_url()
```

Comment:
This should be `get_default_login_return_url()`, as `get_login_return_url()` will check `request.args.get('next')`, potentially allowing the IdP to inject its own redirect parameter. (Ignoring that the IdP could alter RelayState to achieve the same thing).

Both `get_login_return_url()` and `get_default_login_return_url()` can return None in their default implementations. In this case, it would be appropriate to raise an error in `login_successful()`, as there is nothing else we can do. If the developer has not overridden `get_default_login_return_url()` then there is no where to redirect to.

Reply:
Thanks for your review!
As this is a web application, I am a bit reluctant to raise an error "by default". What about returning `/` as default login return URL, so it is not so easy to get a traceback?